### PR TITLE
[NYC-2020] Initial schedule for 2020

### PR DESCRIPTION
**Pull request by shahadarsh**

### CODE REVIEW

File: content/events/2020-new-york-city/program/jameson-hampton.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Example: We’ll go over some real life inventors who have been regretful of the implications of things they’ve built.
+4. Malevolence vs carelessness
+* Not being malevolent isn’t good enough, you have to set up a system that includes protections against other potentially malevolent people.
+5. Thinking about where you can do the most good
+* If you realize that you’re already in a job that you don’t feel good about, we’ll go over the different options you have to deal with that and the pros and cons of finding another job versus trying to affect change from the inside.
+* Example: We’ll also talk about some noteworthy characters who defected from the Empire and how they did so!
+6. Unethical organizations don’t have loyalty to you
+* Agreeing to work for an organization you don’t agree with as a method of self preservation is unwise. They only care about you for as long as you’re useful to them.
+* Example: Again, we’ll talk about the Empire and the lack of job satisfaction involved in trying to please an employer like that.
```
Comment (kuritz) [inline content/events/2020-new-york-city/program/jameson-hampton.md]: Was this outline meant for attendees or only us?

File: content/events/2020-new-york-city/program/michael-wytock.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Versioned Module Library
+Each of your build modules should reference a tagged release of an independently versioned module library. Decoupling the versioning of your build modules from your library modules provides the opportunity for different environments/contexts to use separate versions of your infrastructure components. It also encourages reuse of modules that are present in the module library.
+
+Making Change Easy
+Using the above strategies, multiple environments/deployments can be supported from a common library of modules. Infrastructure development proceeds with adding changes to the module library, tagging those changes, and deploying those changes by referencing the new tag in an infrastructure development environment. If changes succeed, all that changes in the higher environments is the tag referencing your module library.
+
+Because “build modules” are separate from one another, each can reference different versions of the module library. This allows each build module to change independently, if needed.
```
Comment (kuritz) [inline content/events/2020-new-york-city/program/michael-wytock.md]: Did the speaker want all this posted publicly? its a long description

File: content/events/2020-new-york-city/program/tom-limoncelli.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Solution: In this talk, I’ll show how to use the anthropology concept of high-context/low-context societies to eliminate knowledge frustration on DevOps/SRE teams. The result is improved productivity, continuous improvement, and a more enjoyable work environment.
+
+Edward T. Hall was an American anthropologist who coined the phrase “high-context culture,” a culture where there are many unspoken rules. One must know a large amount of context to survive. Low-context cultures have the rules explicitly stated. This model describes engineering teams as well.
+
+This is a talk about how to create a low-context culture where documentation is created incrementally, maintained continuously, and available just in time. Eliminate the “I’ll document it later” syndrome.
+
+Spoiler alert: The bulk of the talk will be tricks to motivate developers and operations engineers to maintain documentation incrementally: Consume wikis with an editor not viewer; start new docs with a template; write checklists, not prose; use the Stack Overflow Enterprise Q&A platform; and more!
```
Comment (kuritz) [inline content/events/2020-new-york-city/program/tom-limoncelli.md]: this says `spoiler alert`, prob shouldn't be posted?

**Comment by mattstratton** [comment]
Is this okay to merge now?

File: content/events/2020-new-york-city/program/chen-harel.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+This session will walk attendees through our open source reliability scoring system in Grafana and a variety of CI/CD tools that provides DevOps teams with a formulaic approach to prioritizing anomalies and understanding the stability of their releases – before they go to production.
+
+The formula scores releases for stability and safety based on if it: – Introduces new errors – Makes existing errors happen at a higher rate (i.e. rate increase) – Introduces slowdowns into the environment
+
+The talk will walk through how to follow the below process and will include an example of the process being applied in practice: – Detect all errors and slowdowns – Classify each event – Prioritize by severity – Score the release – Block release – Visualize the data
```
Comment (kuritz) [inline content/events/2020-new-york-city/program/chen-harel.md]: could be intended to have this outline included, but we might want to remove it if being cautious. hard to tell with it at the end of a thorough description.

**Comment by lizthegrey** [comment]
Please make sure you squish and force push commits to branch to avoid information leakage of the intermediate commits.

**Comment by kuritz** [comment]
@mattstratton i think @shahadarsh wanted to take one more look when he got home from his commute.

**Comment by mattstratton** [comment]
@lizthegrey i always squash on merge, but those things will still show up on the history of the PR?

**Comment by mattstratton** [comment]
Oh wait that was for @shahadarsh

I’m going to tag this “do not merge” for now.

**Comment by shahadarsh** [comment]
Good point @lizthegrey.

**Comment by shahadarsh** [comment]
@mattstratton Good to be merged now.

**Comment by lizthegrey** [comment]
Still leaks. https://github.com/devopsdays/devopsdays-web/pull/9079/commits/1397c39aa38abb21de9e767abd5302d2e4bf2dd2

**Comment by kuritz** [comment]
@shahadarsh if you run...

```
git reset HEAD~2
git add .
git commit -m ...
git push --force
```

it should overwrite the prior 2 commits (unless i messed up 😅 )

**Comment by shahadarsh** [comment]
Thanks. Should be good now.

**Pull request merged**